examples/python-examples/MeshOffsetWeighted.dox.py

```diff
@@ -0,0 +1,27 @@
+from meshlib import mrmeshpy
+
+# Create some mesh
+mesh = mrmeshpy.makeTorus(primaryRadius=1.0,
+                          secondaryRadius=0.1,
+                          primaryResolution=16,
+                          secondaryResolution=16)
+verts_num = mesh.points.size()
+
+# Create VertScalars obj with weights for every vertice
+scalars = mrmeshpy.VertScalars(verts_num)
+for i in range(verts_num):
+    weight = abs(mesh.points.vec[i].x / 5) + 0.1 # Offset size for point
```

Comment:
Better set `weight = abs(mesh.points.vec[i].x / 5)` and `params.offset = 0` below for the same result